<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">.QWidget{background-color: rgb(25,25, 40);}</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="topMargin">
           <number>15</number>
          </property>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">color:rgb(255, 255, 255); font-size:20pt; text-align: center;</string>
            </property>
            <property name="text">
             <string>REGISTER</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="leftMargin">
             <number>50</number>
            </property>
            <property name="topMargin">
             <number>35</number>
            </property>
            <property name="rightMargin">
             <number>59</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="styleSheet">
               <string notr="true">color: rgb(231, 231, 231);
font: 15pt &quot;Verdana&quot;;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>icons/mail_32x32.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="email">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
      color: rgb(231, 231, 231);
      font: 15pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(25, 25, 40);
      selection-background-color: darkgray;
}</string>
              </property>
              <property name="placeholderText">
               <string>Email...</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="Line" name="line">
              <property name="styleSheet">
               <string notr="true">border: 2px solid white;</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">color: rgb(231, 231, 231);
font: 15pt &quot;Verdana&quot;;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>icons/lock_32x32.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="password">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
      color: white;
      font: 15pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(25, 25, 40);
      selection-background-color: darkgray;
}</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="placeholderText">
               <string>Password...</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="Line" name="line_2">
              <property name="styleSheet">
               <string notr="true">border: 2px solid white;</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="styleSheet">
               <string notr="true">color: rgb(231, 231, 231);
font: 15pt &quot;Verdana&quot;;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>icons/lock_32x32.png</pixmap>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="confirmPassword">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>40</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QLineEdit {
      color: rgb(231, 231, 231);
      font: 15pt &quot;Verdana&quot;;
      border: None;
      border-bottom-color: white;
      border-radius: 10px;
      padding: 0 8px;
      background: rgb(25, 25, 40);
      selection-background-color: darkgray;
}</string>
              </property>
              <property name="placeholderText">
               <string>Confirm Password...</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="2">
             <widget class="Line" name="line_3">
              <property name="styleSheet">
               <string notr="true">border: 2px solid white;</string>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="6" column="0" colspan="2">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="8" column="0" colspan="2">
             <widget class="QPushButton" name="registerButton">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>60</height>
               </size>
              </property>
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(231, 231, 231);
font: 17pt &quot;Verdana&quot;;
border: 2px solid orange;
padding: 5px;
border-radius: 3px;
opacity: 200;
</string>
              </property>
              <property name="text">
               <string>Register</string>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="2">
             <widget class="QLabel" name="errorMessage">
              <property name="styleSheet">
               <string notr="true">color: red;
font-size: 18px</string>
              </property>
              <property name="text">
               <string>Error message</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="styleSheet">
             <string notr="true">color: white;
padding-top: 30px;</string>
            </property>
            <property name="text">
             <string>Already have an account?</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loginhereButton">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: orange;
border: 0px;
border-radius: 3px;
opacity: 200;
text-decoration: underline;</string>
            </property>
            <property name="text">
             <string>Login here</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
